examples/batch.py

```diff
@@ -20,16 +20,24 @@
 
 import logging
 
+import docker
 import muselog
 from muselog.logger import get_logger_with_context
 
 from musekafka import consumers
 
 muselog.setup_logging(root_log_level="INFO", module_log_levels={"musekafka": "DEBUG"})
 
+docker_client = docker.from_env()
```

Comment:
I'm a little confused as to the purpose of this? Is it because you're running the example outside of docker but utilizing docker to host the brokers?

Reply:
Yes